lib/core/auth.js

```diff
@@ -234,6 +229,41 @@ export default class Auth {
       })
   }
 
+  requestApollo (operation) {
```

Comment:
I'm not sure but I think it may be better to move this special type of request handler into apollo scheme's class. HTTP (axios) is common enough to be here.

Reply:
I disagree with this.  Graphql is growing in popularity rapidly, and Apollo is the clear front-runner.  

But I appreciate the work you've done getting the library this far and will help where I can.  To be clear, if we did this we could do this   : 

This core file does not have access to the scheme.  So we could change request and requestWith to accept an additional optional parameter for 'requestHandler' and if it has one it invokes it.  If it does not receive a request handler it uses the default with axios.  

Then those implementing custom schemes can implement their login and other methods using a custom handler to use apollo, superaget, or whatever other tools they'd like.

How does that sound?